vsphere/tests/conftest.py

```diff
@@ -5,14 +5,21 @@
 
 import pytest
 from mock import MagicMock, Mock, patch
-from tests.mocked_api import MockedAPI, mock_http_rest_api
+
+from .common import LAB_INSTANCE
+from .mocked_api import MockedAPI, mock_http_rest_api
 
 try:
     from contextlib import ExitStack
 except ImportError:
     from contextlib2 import ExitStack
 
 
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield LAB_INSTANCE
```

Comment:
Note: `dd_environment` change can be moved in another PR. It's here for testing convenience.